keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components/README.md

```diff
@@ -269,17 +270,20 @@ all existing klog features.
 
 #### Alpha
 
-- Klog can be configured without registering flags
-- Kubernetes logging configuration drops global state
+- The remaining supported klog options (`-v`, `--vmodule`) and
+  `--log-flush-frequency` can be configured without registering flags.
+- Kubernetes logging configuration is completely stored in one struct
+  (`LoggingConfiguration`) before being applied to the process.
```

Comment:
As long as we have klog, there is going to be global state in klog (the global logger and its configuration). There's no way around that in alpha and later because this KEP isn't about removing klog.

I discussed this with @serathius in https://kubernetes.slack.com/archives/C020CCMUEAX/p1635445845031900 and he explained that the intention was to treat all logging configuration as local state until applied to the process, so the "stored in one struct" line item is the replacement for "drops global state".